packages/app-store/utils.ts

```diff
@@ -10,6 +10,12 @@ import { appStoreMetadata } from "./apps.browser.generated";
 
 const ALL_APPS_MAP = Object.keys(appStoreMetadata).reduce((store, key) => {
   store[key] = appStoreMetadata[key as keyof typeof appStoreMetadata];
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  //@ts-ignore
+  delete store[key]["/*"];
```

Comment:
Ensure that the props that are never consumed(by party other than app-store-cli) aren't sent.